docs/experience/memory-usage.md

```diff
@@ -0,0 +1,39 @@
+title: Monitoring Browser Memory Usage
+description: Gathering information about web page memory usage
+
+Browsers manage the memory used by web pages automatically without user interaction. Sematext Experience allows monitoring the memory usage for your web pages to identify issues, regressions, and give you an estimate of the memory needed by your applications. 
+
+<img
+  class="content-modal-image"
+  alt="Memory Usage Overview"
+  title="Memory Usage Overview"
+  src="../../images/experience/memory/overview.png"
+/>
+
+## Use Cases
+There are many use cases where having insight into the browser's memory usage can be crucial to identify and fix problems as soon as possible. Those include:
+
+ * Identifying places in your application resulting in high memory usage
+ * A/B testing of memory usage between different application versions
+ * Measuring memory impact of new features or optimizations done to the application code
+ * Statistical analysis of memory usage across the web application
+
```

Comment:
* Identifying memory leaks (in Single Page Applications only?)

Reply:
Added.